core/src/main/java/org/apache/carbondata/core/datamap/dev/DataMapFactory.java

```diff
@@ -110,6 +110,10 @@ public abstract DataMapBuilder createBuilder(Segment segment, String shardName,
    */
   public abstract DataMapLevel getDataMapLevel();
 
+  /**
+   * delete datamap of the segment
```

Comment:
change to `delete datamap data in the specified segment

Reply:
fixed